benchmarks/benchmarks/utils.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
```

Comment:
Is this file copy-pasted (mostly) from the anndata repo?  A lot of these functions don't appear to be used at all.

Reply:
For example, I don't see anywhere `take_view` is used outside of its declaration in this file.